memo/tests/utils.py

```diff
@@ -0,0 +1,5 @@
+import datetime
+
+
+def convertTimeStamp(time):
```

Comment:
the [standard](https://peps.python.org/pep-0008/#function-and-variable-names) for function names in python is to use snake case. can you change this one to `convert_time_stamp`?

Reply:
Okay! I'm not sure why I used snakeCase on those py files. However, I'll use `convert_timestamp` instead of that.